guides/simulcast.md

```diff
@@ -51,43 +28,19 @@ alias Membrane.WebRTC.Extension.{Mid, Rid, TWCC}
   webrtc_extensions: [Rid, Mid, TWCC],
   simulcast_config: %SimulcastConfig{
     enabled: true,
-    default_encoding: fn %Track{simulcast_encodings: _simulcast_encodings} -> "m" end
+    initial_target_variant: fn _track_ -> :medium end
   }
 }
 ```
 
-Here we turn simulcast on and choose medium encoding for each track to be forwarded to the client.
+Here we turn simulcast on and choose medium variant as target for each track
+that is going to be forwarded to the browser.
+Target variant means that it will be choosen whenever it is active.
+If for some reason medium variant is inactive we will temporarily forward some other variant.
```

Comment:
```suggestion
Here we turn simulcast on and choose `:medium` variant as a target for each track
that is going to be forwarded to the browser.
Target variant means that it will be chosen whenever it is active.
If for some reason `:medium` variant is inactive we will temporarily forward some other variant.
```